src/main/java/com/plesba/datamanager/transformers/NullTransformer.java

```diff
@@ -0,0 +1,72 @@
+package com.plesba.datamanager.transformers;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import java.io.IOException;
+import java.io.PipedInputStream;
+import java.io.PipedOutputStream;
+
+public class NullTransformer {
+    private final PipedInputStream inputStream;
+    private PipedOutputStream outputStream;
+    private int recordCount = 0;
+    private byte[] theByteArray = null;
+    private static final Log LOG = LogFactory.getLog(NullTransformer.class);
+
+    public NullTransformer(PipedInputStream parameterInputStream, PipedOutputStream parameterOutputStream ) {
+
+        LOG.info("NullTransformer started processing");
+        inputStream = parameterInputStream;
+        outputStream = parameterOutputStream;
+    }
+
+    public NullTransformer() {
+
+        LOG.info("NullTransformer started processing with no parameters");
+        inputStream = null;
+        outputStream = null;
+    }
+
+    public void processDataFromInputStream() throws IOException {
+
+        try {
+            LOG.info("NullTransformer reading from input stream/writing to output stream");
+
+            StringBuilder recordStringBuffer = new StringBuilder();
+            String streamRecord = new String();
```

Comment:
In Java, strings are immutable, yes? So this initialization allocates a zero-length String that is never used.